notebooks/text_models/solutions/text_generation_using_transformers.ipynb

```diff
@@ -0,0 +1,911 @@
+{
```

Comment:
<p>Maybe give a very brief explanation about what these hyper-parameters are about. In particular, what is the difference between <code>SEQ_LEN</code> and <code>MIN_TRAINING_SEQ_LEN</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Done</p>